flask_testing/utils.py

```diff
@@ -477,7 +478,7 @@ def socket_bind_wrapper(self):
                 return ret
 
             socketserver.TCPServer.server_bind = socket_bind_wrapper
-            app.run(port=port, use_reloader=False)
+            app.run(port=port, use_reloader=False, host=self._configured_host)
 
         self._process = multiprocessing.Process(
             target=worker, args=(self.app, self._configured_port)
```

Comment:
Should probably also add the parameters here:

    self._process = multiprocessing.Process(
        target=worker, args=(self.app, self._configured_host, self._configured_port)

Reply:
And of course https://github.com/jarus/flask-testing/pull/126/files#diff-171a436f5b46787445201095ddb2db3fR463 will have to become

    def worker(app, host, port):